ocaml/sdk-gen/powershell/autogen/XenServerPSModule.psd1

```diff
@@ -41,29 +41,23 @@ CompanyName = 'Citrix Systems, Inc'
 Copyright = 'Copyright (c) Citrix Systems, Inc. All rights reserved.'
 
 # Requirements
-PowerShellVersion = '4.0'
+PowerShellVersion = '7.2'
 PowerShellHostName = ''
 PowerShellHostVersion = ''
-DotNetFrameworkVersion = '4.5'
-CLRVersion = '4.0.30319'
 ProcessorArchitecture = 'None'
 
 #Contents
 ModuleToProcess = 'XenServerPowerShell.dll'
 RequiredModules = @()
 NestedModules = @()
-ModuleList = @('XenServerPowerShell.dll')
-RequiredAssemblies = @('CookComputing.XmlRpcV2.dll',
-                       'Newtonsoft.Json.CH.dll',
+RequiredAssemblies = @('Newtonsoft.Json.CH.dll',

```

Comment:
The latest commit has also removed the ModuleList - was this on purpose or accidental? Is it not needed?

Reply:
It was on purpose. Not only does it seem like it's not needed, but it was flagged by the `Test-ModuleManifest` cmdlet